fillin/src/main/java/com/teamfillin/fillin/domain/film/CompanyEntity.java

```diff
@@ -7,7 +7,7 @@
 import javax.persistence.Id;
 
 @Entity
-public class FilmType {
+public class CompanyEntity {
```

Comment:
```suggestion
@Table(name = "company")
public class CompanyEntity {
```
전체적으로 Entity postfix 가 붙으면서 데이터베이스에 테이블 명이 ~Entity로 매핑될 것 같아서, @Table 로 name 명시해주는게 좋을 것 같아요!